src/mono/mono/mini/interp/transform.c

```diff
@@ -3737,12 +3737,29 @@ get_basic_blocks (TransformData *td, MonoMethodHeader *header, gboolean make_lis
 
 	for (guint i = 0; i < header->num_clauses; i++) {
 		MonoExceptionClause *c = header->clauses + i;
+		if (start + c->try_offset <= 0) {
```

Comment:
The pointer arithmetic around here looks redundant. If you really want to do checks for these offsets, it would make more sense to check them against the code_size directly. Note that all offsets in `MonoExceptionClause` are unsigned. This comparison that a pointer is negative doesn't make sense to me, I don't think this code should compile.

Reply:
Make sense. As offsets are unsigned, only checks against the `code_size` are in place.